tests/conformance/README.md

```diff
@@ -119,4 +119,63 @@ If you want to combine VS Code & dlv for debugging so you can set breakpoints in
     },
     ]
 }
+```
+
+## Using terraform for conformance tests
+
+If you are writing new conformance tests and they require cloud resources, you should use the 
+terraform framework we have in place. To enable your component test to use terraform there are a few changes in the normal steps you must do.
+
+1. In the `conformance.yml` you should create a new step in a workflow for your component that creates new env variables. You will need a variable for each specific resource your tests will use. If you require 3 different topics and 2 different tables for your tests you should have 5 different env variables set. The only convention you must follow for the variables is the value must use `env.UNIQUE_ID` to ensure there are no conflicts with the resource names.
+
+```bash
+PUBSUB_AWS_SNSSQS_QUEUE="testQueue-${{ env.UNIQUE_ID }}"
+echo "PUBSUB_AWS_SNSSQS_QUEUE=$PUBSUB_AWS_SNSSQS_QUEUE" >> $GITHUB_ENV
+```
+
+2. When updating the `tests.yml` defined inside `tests/config/<COMPONENT-TYPE>/` folder you should overwrite the default names of any resources the conformance tests use. These values should reference env variables which should be defined in the conformance.yml.
+
+```yaml
+- component: aws.snssqs.terraform
+    operations: ["publish", "subscribe", "multiplehandlers"]
+    config:
+    pubsubName: aws-snssqs
+    testTopicName: ${{PUBSUB_AWS_SNSSQS_TOPIC}}
+    testMultiTopic1Name: ${{PUBSUB_AWS_SNSSQS_TOPIC_MULTI_1}}
+    testMultiTopic2Name: ${{PUBSUB_AWS_SNSSQS_TOPIC_MULTI_2}}
+```
+
+3. When writing your `component.yml` you should reference your credentials using env variables and any resources specified in the yaml should use env variables as well just as you did in the `test.yml`. Also
+`disableEntityManagement` should be set to `true` since we want to use only terraform to provision resources.
```

Comment:
Not every component even has entity management, so it is best if you explain what this is and that this option might be called something else, but should be deactivated.